src/show/pages/public/Standings/index.jsx

```diff
@@ -1,161 +1,37 @@
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '3',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '4',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '5',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '6',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '7',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '8',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '9',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '10',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  },
-
-  {
-    id: '11',
-    name: 'Charles Leclerc',
-    points: 157,
-    place: 1,
-    avatar_url:
-      'https://www.formula1.com/content/dam/fom-website/drivers/C/CHALEC01_Charles_Leclerc/chalec01.png.transform/2col/image.png'
-  }
-];
-
 const Standings = () => {
+  const [constructor, setConstructor] = useState([]);
+  const [driver, setDriver] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get('http://localhost:3000/v1/constructors/')
+      .then(res => setConstructor(res.data))
+      .catch(err => err);
+  }, []);
+
+  useEffect(() => {
+    axios
+      .get('http://localhost:3000/v1/drivers/')
+      .then(res => setDriver(res.data))
+      .catch(err => err);
+  }, []);
```

Comment:
Since this is using the same use effect dependencies both axios fetches can be done in the same useEffect.